### ISSUE TITLE
building error

### ISSUE THREAD

**Issue by Ruslan0Dev** [kaitai-io/kaitai_struct_gui#14]
I tried to compile but got the error:
![image](https://user-images.githubusercontent.com/23433847/127788491-77ddbdd5-9c5a-4b45-87dd-a94f74dec4cd.png)

**Comment by generalmimon** [comment]
Cc @Mingun

**Comment by GreyCat** [comment]
With sunset of bintray, it looks like we'll have to republish JHexView to some other place... Or, better yet, have somebody take ownership of that code and start publishing that to proper public repos (e.g. Central Maven).

**Comment by Mingun** [comment]
> Or, better yet, have somebody take ownership of that code and start publishing that to proper public repos (e.g. Central Maven).

  I think that would be the best option. I can't find any way to publish an artefact without gpg magic and very confusing guides. So if you already have working account to publish to Maven Central I'll be glad to provide you all required access. Otherwise I have to learn gpg and maven publish plugins...

**Comment by GreyCat** [comment]
@Mingun Central Maven's access is granted on per-domain org, so we've got one registered for io.kaitai (given that we own https://kaitai.io/).

To publish JHexView using that, at bare minimum JHexView code has to be refactored to use io.kaitai namespace - so, effectively, we're talking about a fork of https://github.com/sporst/JHexView ?

**Comment by dgelessus** [comment]
> To publish JHexView using that, at bare minimum JHexView code has to be refactored to use io.kaitai namespace

I don't think the code needs to be changed - it doesn't contain references to any specific Maven group ID, right? The library's *package* is called tv.porst.jhexview, but you can still upload it under the io.kaitai group - there's no requirement that the Java package name matches the Maven group ID.

**Comment by Mingun** [comment]
> To publish JHexView using that, at bare minimum JHexView code has to be refactored to use io.kaitai namespace - so, effectively, we're talking about a fork of https://github.com/sporst/JHexView ?

No, as @dgelessus said, you can just republish project under the new group id. If you need my assistance as JHexView project owner (because this fork is used in this project) to integrate these changes into the repository, I would glade to cooperate.

I also want to note to everyone interested that the further development of this project is difficult without the adoption of https://github.com/kaitai-io/kaitai_struct_compiler/pull/191, https://github.com/kaitai-io/kaitai_struct_java_runtime/pull/28 and related.
